include/fi_util.h

```diff
@@ -602,11 +602,6 @@ int ofi_eq_create(struct fid_fabric *fabric, struct fi_eq_attr *attr,
  */
 #define OFI_MR_BASIC_MAP (FI_MR_ALLOCATED | FI_MR_PROV_KEY | FI_MR_VIRT_ADDR)
 
-#define OFI_CHECK_MR_BASIC(mode) ((mode == FI_MR_BASIC) || \
-				  ((mode & OFI_MR_BASIC_MAP) == OFI_MR_BASIC_MAP))
-
-#define OFI_CHECK_MR_SCALABLE(mode) (!(mode & OFI_MR_BASIC_MAP))
```

Comment:
Support for FI_MR_BASIC and FI_MR_SCALABLE are explicitly indicated by the provider as a supported mr_mode